src/test/java/org/springframework/cloud/deployer/scheduler/spi/kubernetes/ImagePullPolicyTests.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.deployer.scheduler.spi.kubernetes;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for {@link ImagePullPolicy}.
+ *
+ * @author Chris Schaefer
+ */
+public class ImagePullPolicyTests {
```

Comment:
this might shuffle elsewhere if we go with a single `ImagePullPolicy` enum rather than dups